app/Models/BoardCat.php

```diff
@@ -43,11 +43,11 @@ class BoardCat extends Model
 
     protected $guarded = [];
 
-    public function threads(){
+    public function threads() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->